README.md

```diff
@@ -0,0 +1,22 @@
+# Additional Fruits
+This module adds various different fruit blocks and includes 3 different stages for each plant.
+
+**Types of Fruits:**
+* Blueberry
+* Bushes
+  - Cranberry
+  - Gojiberry
+  - Guava
+  - Peach
+* Gooseberry
+* Raspberry
+* Strawberry
+
+
+How to Contribute
+ ============
+ 1. Fork and clone this repository locally
+ 2. To add another **fruit block** or **fruit bush**, create a new prefab
+ 3. Add a texture to your fruit, a PNG file is reccomended
```

Comment:
In fact PNG is required :-) And ideally of ^2 dimensions (16x16, 32x32..) - I believe there are more existing details on the topic you can find and link to

Additionally it is important to consider attribution and licensing. Not just any image will work - the author (and us as project maintainers) need to be authorized to use the assets. If the author creates their own image they should claim credit so we can attribute them. This is another topic that is covered some in similar modules and on some wiki pages. Try to find them - if you can't then ask a mentor :-)